cli/secret/secret_set.go

```diff
@@ -46,11 +46,11 @@ var secretUpdateCmd = &cli.Command{
 			Usage: "secret value",
 		},
 		&cli.StringSliceFlag{
-			Name:  "events",
+			Name:  "event",
 			Usage: "secret limited to these events",
 		},
 		&cli.StringSliceFlag{
-			Name:  "images",
+			Name:  "image",
 			Usage: "secret limited to these images",
```

Comment:
Not related to this PR but the `Usage` seems to be a copy/paste one and is wrong.

Reply:
Ah no... My bad